manifests/config.pp

```diff
@@ -47,7 +47,7 @@
     }
   }
 
-  if $::vault::install_method == 'archive' {
+  if $::vault::install_method == 'archive' or $::vault::manage_service_file {
```

Comment:
What if I want to install via archive but do **not** want to use the service file provided by this module? Seems like this configuration isn't possible with this code.